app/Instructor/attendance/Config.php

```diff
@@ -0,0 +1,22 @@
+<?php
```

Comment:
@daxpatel1 this is establishing a DB connection to your locally hosted MySQL database. It won't work when the app is deployed to AWS. 

In fact, you don't even need to include `app/config.php` as it's already included in `header.php`:
https://github.com/CIS101-PaceU/infrastructure/blob/develop/app/header.php

You should not have a `config.php` in your attendance folder

Reply:
@wdickerson I need a query that needs to be working in both my pages thats why I had my on config file.

if you want me to link your config then I have to add that query in both the pages. 

So if you could please add the aws credentials to my config file as well.